geotrellis-sentinelhub/src/main/scala/org/openeo/geotrellissentinelhub/PyramidFactory.scala

```diff
@@ -233,6 +233,9 @@ class PyramidFactory(collectionId: String, datasetId: String, catalogApi: Catalo
   Seq[(Int, MultibandTileLayerRDD[SpaceTimeKey])] = {
     // TODO: use ProjectedPolygons type
     // TODO: reduce code duplication with pyramid_seq()
+    if (dataCubeParameters.timeDimensionFilter.isDefined) {
+      throw new IllegalArgumentException("OpenEO does not support timeDimensionFilter yet for SentinelHub layers. (Probably used in filter_labels)")
```

Comment:
@jdries Error message ok?

Reply:
I don't think user will know what timeDimensionFilter is, and 'layer' is also not a user-facing concept. 
I tend to write errors like:
'filter_labels on time dimension is not supported for sentinelhub collections.'

Reply:
Changed: https://github.com/Open-EO/openeo-geotrellis-extensions/commit/0d1025843435f87337111a6ba2406e40e6619676